src/kpatch_log.h

```diff
@@ -3,6 +3,18 @@
 
 #include <stdio.h>
 
+#define	ERROR_SUCCESS		0	// exit code 0
+#define	ERROR_GENERAL		-255	// exit code 1
```

Comment:
Let's introduce a macro named `ERR_CODE(x) (x - 256)`.
and use it here. For instance:
```
#define ERROR_GENERAL ERR_CODE(1) /* general error */
```